requirements.txt

```diff
@@ -9,7 +9,7 @@ emails
 fastapi-caching[redis]
 fastapi-pagination[sqlalchemy]~= 0.9.3
 fastapi[all]==0.78.0
-fideslang==1.0.0
+fideslang==1.1.0
```

Comment:
🎉 
For my own understanding, can you explain why we're moving away from the provided/derived and identifiable/non-identifiable categories?